src/content/docs/tutorial-diagnose-infra-bottlenecks/diagnose-app-infra-data.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: Diagnose app failure with host data
+metaDescription: Learn how to identify failing hosts and apps using infrastructure data.
+---
+
+import alertSummaryPage from 'images/infrastructure_screenshot-full_hosts-alert-critical.webp'
+
+import tableSummaryPage from 'images/infrastructure_screenshot-full_table-ordered.webp'
+
+import serviceMapHosts from 'images/infrastructure_screenshot-service-maps-app.webp'
+
+import findApp from 'images/infrastructure_screenshot-find-app.webp'
+
+import filterApp from 'images/infrastructure_screenshot-full_hosts-filtered-app.webp'
+
+import hostSummaryPage from 'images/infrastructure_screenshot-full_hosts-summary-page.webp'
+
+import metrics1 from 'images/infrastructure_screenshot-full_metrics-time1.webp'
+
+import metrics2 from 'images/infrastructure_screenshot-full_metrics-time2.webp'
+
+import summaryTableAlerting from 'images/infrastructure_screenshot-full_summary-table-alerting.webp'
+
+When your system is fully instrumented, you can correlate data between your system infrastructure and the apps your infrastructure supports. It's likely, though, that you have thousands of faceless hosts allocating resources to various apps. You may not have the full context of what is happening where, which makes finding relevant data overwhelming. How do you sort through all of your data to find infrastructure-related causes to failing apps?
+
+## Objectives [#objectives]
+
+This doc guides you through finding relevant data within the infrastructure UI. You will:
+
+* Filter your infrastructure data by attributes
+* Identify specific hosts and apps without additional context
+* Use the timepicker to find when a change occurred 
+
+## Explore your host data to find the cause of an outage [#explore]
+
+<Steps>
+    <Step>
+    ### Identify failing hosts
+
+<Tabs>
+	<TabsBar>
+        <TabsBarItem id="1">
+            Find hosts with alerts
+        </TabsBarItem>
+        <TabsBarItem id="2">
+            Find hosts without alerts
+        </TabsBarItem>
+    </TabsBar>
+
+    <TabsPages>
+        <TabsPageItem id="1">
+    If you're uncertain about how to get started, we recommend initially scoping your hosts by alert severity. Using the summary page overview, you can see that there are three critical alert violations happening in your system.
```

Comment:
Do you already have to have set up alerts for this to work? It's the apps that _contain_ hosts with alert severity `critical` that are causing the alert signal, right?